include/juice/juice.h

```diff
@@ -90,6 +90,12 @@ typedef enum juice_concurrency_mode {
 	JUICE_CONCURRENCY_MODE_THREAD,   // Each connection runs in its own thread
 } juice_concurrency_mode_t;
 
+typedef enum juice_ice_tcp_mode {
+	JUICE_ICE_TCP_MODE_NONE = 0, // ICE-TCP is disabled
+	JUICE_ICE_TCP_MODE_ACTIVE,   // ICE-TCP will operate as a client
+	JUICE_ICE_TCP_MODE_PASSIVE,  // ICE-TCP will operate as a server
```

Comment:
As I understand TCP candidates are not exclusive. An agent may gather both active, passive (and possibly simultaneous open) candidates.

Reply:
They can be! I believe browsers only do active TCP. They don't accept inbound TCP connections.